lxd/container_backup.go

```diff
@@ -123,6 +123,11 @@ func containerBackupsPost(d *Daemon, r *http.Request) Response {
 		return BadRequest(fmt.Errorf("Backup names may not contain slashes"))
 	}
 
+	// Add a default expiry time of 30 minutes
+	if req.ExpiryDate.IsZero() {
```

Comment:
How does one make a backup which doesn't expire?

I think the best here would be to set req.ExpiryDate to 30min from now BEFORE we parse the user request (unmarshal json), that way if the JSON data doesn't include the expiry_date field, they'll get 30min but if it is in the JSON and is empty, then they get a non-expiring backup.

Reply:
Sounds good. However, we should use something like `"expiry" = "0"` if we want to disable backups since we cannot always distinguish between an empty string and an omitted field.